package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "mocha": "^8.3.2",
     "puppeteer": "^8.0.0",
+    "stdio": "^2.1.1",
```

Comment:
Unfortunately, I cannot reproduce the "npm ci" issue when your PR is build (https://github.com/larsthorup/mocha-vite-puppeteer/pull/8/checks?check_run_id=2978430300). 

But the error indicates that you have added the "stdio" package, as reflected in "package.json", and that is not reflected in "package-lock.json". 

Could you run "npm install" locally and include the updated "package-lock.json" in this PR. Hopefully that will make the build green 🚀